ldk/javascript/examples/self-test-loop/src/tests/browser/index.ts

```diff
@@ -198,3 +204,30 @@ export const testListenNavigationType = (): Promise<boolean> =>
       reject(error);
     }
   });
+
+export const testListenUIElement = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    try {
+      const url = 'https://www.oliveai.dev';
+      await browser.openTab(url);
+      const UIArguments = {
```

Comment:
Nitpick, but you don't normally name variables with a capital letter first.

Reply:
@BrettMugglinOlive  Good catch!